robottelo/ui/dashboard.py

```diff
@@ -10,21 +10,6 @@
 class Dashboard(Base):
     """Manipulates dashboard from UI"""
 
-    widgets_locators = {
```

Comment:
Now `@data-name` has same value as displayed widget name, so this workaround is not needed.